README.rst

```diff
@@ -122,3 +122,16 @@ License
 -------
 
 The project is licensed under the GNU GPLv3 license.
+
+
+Build the Package
+=================
+Run the following command in the terminal to build the package:
+```bash
+python setup sdist
+```
+This command will create a dist/ directory with a source distribution of your library.
+Install the local package using pip:
+```bash
+pip install dist/quiffen-1.0.tar.gz
+```
```

Comment:
This is incorrect. Quiffen uses Poetry, so the correct build command is `poetry build`.

Reply:
Also, I'm not sure this is in the scope of this PR. If you'd like to make these changes, please open another PR.